kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue

```diff
@@ -83,13 +128,56 @@
       optionIconColor() {
         return this.active ? this.$themeTokens.primary : null;
       },
+      iconAfter() {
+        return this.visibleSubMenu ? 'chevronUp' : 'chevronDown';
+      },
+    },
+    mounted() {
+      this.submenuShouldBeOpen();
     },
     methods: {
       conditionalEmit() {
         if (!this.link) {
           this.$emit('select');
         }
       },
+      isActiveLink(route) {
+        return route.includes(this.$router.currentRoute.path);
+      },
+      submenuShouldBeOpen() {
+        // which plugin are we currently in?
+        if (this.subRoutes) {
+          const key = Object.keys(this.subRoutes)[0];
+          this.subRoutes[key].route.includes(window.location.pathname)
```

Comment:
This might lead to false positives in the case of say, Learn, where MyDownloads is nested under the learn plugin, but not at the top level.

We need to ensure an exact match of the pathname rather than `includes`, as your use of the pathname has already excluded any query parameters and the hash.